scipy-notebook/Dockerfile

```diff
@@ -6,9 +6,9 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
 USER root
 
-# libav-tools for matplotlib anim
+# ffmpeg for matplotlib anim
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends libav-tools && \
+    apt-get install -y --no-install-recommends ffmpeg && \
```

Comment:
Noting this resolves https://github.com/jupyter/docker-stacks/issues/626

Reply:
Think this resolves #611 rather than #626

Reply:
Indeed. Wrong #